nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -13,7 +13,12 @@ let
   phpPackage = cfg.phpPackage.buildEnv {
     extensions = { enabled, all }:
       (with all;
-        enabled
+        # disable default openssl extension
+        (lib.filter (e: e != cfg.phpPackage.extensions.openssl) enabled)
```

Comment:
Can't we filter out by the package's pname for instance? This would also make sure that overlayed `php.extensions.openssl` is correctly filtered out.

Reply:
We definitely can